samples/winapi_filefind/main.c

```diff
@@ -15,6 +16,17 @@ int main()
 
     pb_show_debug_screen();
 
+    // Mount C:
+    OBJECT_STRING MountPath = RTL_CONSTANT_STRING("\\??\\C:");
+    OBJECT_STRING DrivePath = RTL_CONSTANT_STRING("\\Device\\Harddisk0\\Partition2\\");
+    NTSTATUS status;
+    status = IoCreateSymbolicLink(&MountPath, &DrivePath);
+    if (!NT_SUCCESS(status)) {
+        debugPrint("Failed to mount C: drive!\n");
+        XSleep(5000);
+        return -1;
+    }
+
```

Comment:
Should there be some helper API which does mounting for compatibility with old code or MS XAPI?

It could be called from CRT0 (but it would have to be optional, to keep side-effects low for low-level use-cases).

---

Even if we decide against that, consider moving it into a separate function, so users can copy/paste this part to their own projects more easily.

Reply:
Imo there should be an API for that, but I haven't found a good fit yet. There are Windows APIs for similar functionality but those are bit different in what they actually do/expect, and I didn't see a candidate in XAPI either - in fact, the only code I've seen written against XDK that used something like this just used `IoCreateSymbolicLink` directly, too.
The fact that we don't have something like that yet is part of the reason why I'm not axing the X*-functions with this PR already.

I just pushed a change that moves the mounting in a separate function in the sample, though.